catalog/transformer/catalog-transformer-streaming-impl/pom.xml

```diff
@@ -88,13 +92,17 @@
                         <Bundle-SymbolicName>${project.artifactId}</Bundle-SymbolicName>
                         <Bundle-Name>${project.name}</Bundle-Name>
                         <Embed-Dependency>
+                            catalog-transformer-streaming-dependencies;inline=true,
                             catalog-core-api-impl,
                             ogc-tools-gml-jts,
                             gml-v_3_1_1-schema,
                             jaxb2-basics-runtime,
                             catalog-transformer-streaming-lib,
                             platform-util
                         </Embed-Dependency>
+                        <Import-Package>
+                            *;resolution:=optional
```

Comment:
❓ Won't this hide unknown issues until run time if one of the bundle's imports are not satisfied?

Reply:
You are correct, this was just the quickfix. Pushing change that has an explicit list.